README.md

```diff
@@ -70,50 +82,70 @@ The default template uses JUnit 5. You may be used to JUnit 4, but here are some
 
-1.  Get the [odo v.1.2.6](https://mirror.openshift.com/pub/openshift-v4/clients/odo/v1.2.6/) CLI (or later).
+1.  Install the latest vesion of OpenShift Do (odo) using [these instructions](https://odo.dev/docs/installing-odo/)
 
-1. Odo must be run in experimental mode at the 1.x level in order to access the devfile support. Enable experimental mode using the odo CLI:
+    <!-- Keep for now in case future changes require experimental mode
+    1. Odo must be run in experimental mode at the 1.x level in order to access the devfile support. Enable experimental mode using the odo CLI:
     ```shell
     odo preference set experimental true
     ```
+    --->
 
 1. Create a new folder in your local directory and create an odo project using the Odo CLI, e.g.:
     ```shell
     mkdir my-project
     cd my-project
-    odo project create <project namespace name> 
+    odo project create <project namespace name>
     ```
     The project namespace name will be used to create an openshift namespace that will house your application and related pods
 
-1. Initialize the local folder for the odo project
+1. Initialize the local folder using the Open Liberty stack.
     ```shell
-    odo create java-openliberty <app project name> --starter
+    odo create java-openliberty --starter
     ```
-    
+
     This will initialize an Open Liberty project and download the default starter app artifacts.
 
-1. Create an odo URL that will be used to access the starter app:
-    ```shell
-    odo url create --port 9080
-    ```
-1. Once your project has been initialized and the URL created, you can push your application to your openshift cluster for the first time using the following command:
+1. Once your project has been initialized, you can push your application to your openshift cluster for the first time using the following command:
 
     ```shell
-    odo push 
+    odo push
     ```
 
     Upon a first time `odo push` invocation, the following occurs:
-    1. An Open Shift managed kubernetes pod is launched within the namespace created in step 3 above, 
-    1. A container based on the image referenced in the devfile is started in that pod 
-    1. odo syncs your local source code directory to that container. (under the `/projects` directory)
-    1. The app is compiled and built within the container and liberty dev mode is started to host the app once built.
-    1. The URL is activated and is usable to access the app 
+    - An Open Shift managed kubernetes pod is launched within the namespace created in step 3 above,
+    - A container based on the image referenced in the devfile is started in that pod
+    - odo syncs your local source code directory to that container. (under the `/projects` directory)
+    - The app is compiled and built within the container and liberty dev mode is started to host the app once built.
+    - A URL is activated and is usable to access the app
+
+1. (Optional) Create a secure URL
```

Comment:
My thinking was that most of the interaction from a developer would be from their local workspace. So having them access the container to hit a url might be against the expected flow?  Accessing metrics is one example, but maybe they are trying out other security features for their app and need to use SSL? 

For the minikube vs OCP, I'd say we stick with OCP for now since its the main path. We could do a separate section for minikube or a separate article if we see it being a common use case.

Reply:
Fine ignoring minikube.

I think the secure port is a bit too ugly with all the workarounds at the moment.   Can we just move this to a separate docs page for now?